src/migrations/20210828065453_daily_player_stats_table.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function (knex) {
+    return knex.schema.createTableIfNotExists("daily_player_stats", function (table) {
+        table.string("player_id").notNullable();
+        table.integer("songs_guessed").defaultTo(0);
+        table.integer("games_played").defaultTo(0);
+        table.integer("exp").defaultTo(0);
+        table.integer("level").defaultTo(0);
```

Comment:
What's the purpose of recording level gains? Doesn't scale linearly

Reply:
It displays the levels gained. It is still a nice to have secondary stat in the leaderboard